libs/common/src/billing/models/response/organization-subscription.response.ts

```diff
@@ -11,7 +11,7 @@ export class OrganizationSubscriptionResponse extends OrganizationResponse {
   subscription: BillingSubscriptionResponse;
   upcomingInvoice: BillingSubscriptionUpcomingInvoiceResponse;
   expiration: string;
-
+  selfHostSubscriptionExpiration: string;
```

Comment:
Nit: missing a line break between this line and the constructor definition

```suggestion
  selfHostSubscriptionExpiration: string;

```